app/bundles/CoreBundle/Controller/ThemeController.php

```diff
@@ -199,12 +199,36 @@ public function deleteAction(Request $request, ThemeHelperInterface $themeHelper
     public function batchDeleteAction(Request $request, ThemeHelperInterface $themeHelper)
     {
         $flashes = [];
+        $error   = null;
```

Comment:
```suggestion
        $error   = [];
```
`$error` should be an array as it is expected on line 211.

Reply:
done